AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt

```diff
@@ -9,19 +9,21 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.AdapterView
 import android.widget.ArrayAdapter
+import android.widget.ListView
 import android.widget.TextView
 import com.applovin.enterprise.apps.demoapp.R
 import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem
-import kotlinx.android.synthetic.main.activity_list.*
 
 
 abstract class DemoMenuActivity : AppCompatActivity()
 {
+    protected lateinit var list_view: ListView
```

Comment:
Let's match Java's naming here. 
```suggestion
    protected lateinit var listView: ListView

```

Reply:
@jplianghu - should look around the codebase on our conventions

Reply:
For example if it's lowerCamelCase everywhere, and you use snake case, and if everyone starts using their own convention, the code becomes unreadable very fast.

Reply:
For layouts we don't really have a convention, so you may see snake_case there.